pkgs/by-name/gx/gxml/package.nix

```diff
@@ -0,0 +1,54 @@
+{ fetchFromGitLab
+, glib
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, stdenv
+, lib
+, libgee
+, libxml2
+, vala
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gxml";
+  version = "0.20.3";
+
+  outputs = [ "out" "dev" "devdoc" ];
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "gxml";
+    rev = finalAttrs.version;
+    hash = "sha256-GlctGxsLyQ2kPV3oBmusRiouG4PPncBTh3vgxhVaQOo=";
+  };
+
+  doCheck = true;
+
+  nativeBuildInputs = [
+    meson
+    pkg-config
+    ninja
+    vala
+    gobject-introspection
+  ];
+
+  propagatedBuildInputs = [
+    libxml2
+    glib
+    libgee
+  ];
+
+  postPatch = ''
+    substituteInPlace gxml/gxml.pc.in --replace-fail "@prefix@/include" "@includedir@"
```

Comment:
I doubt if this actually works (did you test building planify 4.5?), I don't see `includedir` being set in [configuration](https://gitlab.gnome.org/GNOME/gxml/-/blob/0.20.3/gxml/meson.build?ref_type=tags#L6) :

On 4b35466df18438d24b3e6a23cb783170a352975c:

```
$ cat $(nix-build -A gxml.dev)/lib/pkgconfig/gxml-0.20.pc | grep includedir
includedir=
Cflags: -I${includedir}/gxml-0.20
```

I pushed some small fixups for this, I think replacing this with `${placeholder "dev"}/include` should be good